.gitignore

```diff
@@ -1,4 +1,3 @@
+dist
 node_modules
-.env
-messaging_config.json
-messaging_store.json
\ No newline at end of file
+data
```

Comment:
The only setting that actually includes is tab width, which I feel like is pretty helpful since Prettier is also configured to have tab size be 2. I can remove it if you'd like.